app/src/androidTest/java/com/github/ybecker/epforuml/CoursesFragmentTest.kt

```diff
@@ -89,8 +89,9 @@ class CoursesFragmentTest {
         onView(withId(R.id.nav_courses)).perform(click())
 
         ClickOnSwitch(checkPosition)
-
-        assertTrue(db.getUserSubscriptions(user).size == 1)
+        db.getUserSubscriptions(user.userId).thenAccept {
+            assertTrue(it.size == 1)
```

Comment:
Maybe you should give some details on what you are testing, e.g. : what is this "1+ for?